app/insituIndicateurs/lib.ts

```diff
@@ -171,3 +198,19 @@ export const mappingsIsReady = (mappings: WidgetColumnMap | null) => {
     mappings[COLUMN_MAPPING_NAMES.MAILLE.name]
   );
 };
+
+export const removeAccents = (str: string): string => {
+  return str.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
+};
+
+export const listObjectToString = (objList: object[]): string => {
+  let text = "";
+  objList.forEach((row: object, index: number) => {
+    text += "{";
+    for (const [key, value] of Object.entries(row)) {
+      text += key + ": " + value + ", ";
+    }
+    text += index - 1 === objList.length ? "}, " : "}";
+  });
+  return text;
```

Comment:
Maybe shorter and simpler?

```suggestion
  return objList.map((row) => {
    return `{${Object.entries(row).map([key, value] => `${key}: ${value}`).join(", "))}}`
  }).join(", ");
```